packages/fxa-auth-server/lib/senders/emails/templates/verifyAccountChange/index.txt

```diff
@@ -8,6 +8,6 @@ verifyAccountChange-prompt = "If yes, here is your authorization code:"
 
 <%- code %>
 
-verifyAccountChange-expiry-notice = "It expires in <%- expirationTime == 1 ? '1 minute' : expirationTime + ' minutes' %>."
+verifyAccountChange-expiry-notice = "It expires in <%- expirationTime === 1 ? '1 minute' : expirationTime + ' minutes' %>."
```

Comment:
Does this not need a new ID since it hasn't been pulled over to pontoon yet?

Reply:
No. The file failed to parse initially, I fixed it on our side now and it's now exposed. This is just mirroring the fix I already made.